packages/swing-store/src/snapStore.js

```diff
@@ -36,13 +45,30 @@ function fail() {
   assert.fail('snapStore not available in ephemeral swingStore');
 }
 
+/**
+ * This is a polyfill for the `buffer` function from Node's
+ * 'stream/consumers' package, which unfortunately only exists in newer versions
+ * of Node.
+ *
+ * @param {AsyncIterable<Buffer>} inStream
+ */
+export const buffer = async inStream => {
+  const chunks = [];
+  for await (const chunk of inStream) {
+    chunks.push(chunk);
+  }
+  return Buffer.concat(chunks);
+};
+
 /** @type {SnapStore} */
 export const ephemeralSnapStore = {
-  has: fail,
-  load: fail,
-  save: fail,
-  prepareToDelete: fail,
-  commitDeletes: fail,
+  hasHash: fail,
```

Comment:
You should be able to delete `ephemeralSnapStore`, and the `fail` function, since all swingstores will now include fully-functional snapstores.

(I think eslint didn't complain because it was exported, so it doesn't know for sure that nobody else will use it)